src/networking/payload/payload.rs

```diff
@@ -0,0 +1,17 @@
+use super::packet::Packet;
+
+
+
+
+
+pub trait Ipv8Payload {
+
+  // // fn serialize(&self) -> Vec<PackingIdentifier>;
+  // fn deserialize(data:Vec<PackingIdentifier>) -> Self;
+
```

Comment:
Please remove this unnecessary empty line

Reply:
whoops, forgot to remove those comments